frontend/src/App.js

```diff
@@ -1,20 +1,24 @@
 import React from 'react';
```

Comment:
Great job. You should  be proud. Get yourself a cup of coffee :)

Reply:
Thanks :)